### PR TITLE
Issue/128 povgen

### PR DESCRIPTION
**Pull request by vitalych**

### CODE REVIEW

File: s2e_env/commands/forkprofile.py [hunk 1]
```diff
@@ -75,7 +75,8 @@ def get_profile(self):
             for rel_pc, count in counts.iteritems():
                 try:
                     sym, fcn = self._syms.get(mod.path, rel_pc)
-                except Exception:
+                except Exception as e:
+                    logger.debug(e, exc_info=1)
```
Comment (adrianherrera) [inline s2e_env/commands/forkprofile.py]: Should these be going to a different stream? E.g. `warning`

**Comment by adrianherrera** [comment]
Looks like this branch needs to be rebased with master

File: s2e_env/commands/forkprofile.py (hunk 1 above)
Comment (vitalych) [reply s2e_env/commands/forkprofile.py]: No, the exception is normal, it would make it too verbose.

Comment (adrianherrera) [reply s2e_env/commands/forkprofile.py]: ACK

**Comment by adrianherrera** [comment]
Looks like it needs a rebase

**Pull request merged**